qa/tasks/vstart_runner.py

```diff
@@ -416,6 +416,7 @@ def run(self, **kwargs):
     def _do_run(self, args, check_status=True, wait=True, stdout=None,
                 stderr=None, cwd=None, stdin=None, logger=None, label=None,
                 env=None, timeout=None, omit_sudo=True, shell=True):
+        omit_sudo = False
```

Comment:
This is to make `qa/tasks/cephfs/test_nfs.py` work, right? I think the patch required is:

````
diff --git a/qa/tasks/cephfs/test_nfs.py b/qa/tasks/cephfs/test_nfs.py
index 3013dffe1e5..e42c3dab0d1 100644
--- a/qa/tasks/cephfs/test_nfs.py
+++ b/qa/tasks/cephfs/test_nfs.py
@@ -256,7 +256,7 @@ class TestNFS(MgrTestCase):
         '''
         try:
             self.ctx.cluster.run(args=['sudo', 'mount', '-t', 'nfs', '-o', f'port={port}',
-                                       f'{ip}:{pseudo_path}', '/mnt'])
+                                       f'{ip}:{pseudo_path}', '/mnt'], omit_sudo=False)
         except CommandFailedError as e:
             # Check if mount failed only when non existing pseudo path is passed
             if not check and e.exitstatus == 32:
````

and elsewhere.